gxa/src/main/webapp/WEB-INF/jsp/tiles/views/search-results/bioentity-information.jsp

```diff
@@ -14,8 +14,8 @@
 <script>
     document.addEventListener("DOMContentLoaded", function(event) {
         expressionAtlasBioentityInformation.render({
-            target: 'bioentityInformationTab',
-            payload: ${bioentityProperties}
-        })
+            atlasUrl: "${pageContext.request.contextPath}/",
+            geneId: "${gene_id}"
```

Comment:
Ahh I just realised where you are using the `gene_id` that you added in the `BioentityCardModelFactory`. I think you can pass the gene ID from an attribute that is already available. This `bioentity-information.jsp` is included in the `search-results.jsp`, and that page should already know what gene ID is being displayed.

Reply:
You are right that an attribute (entityFullName) already exists in search_results.jsp but the problem with that is it combines identifier and entityName together. So to just get identifier in the model I added a new field.

Reply:
I'm fairly sure the `identifier` attribute will give you what you want.

Reply:
I was referring to the `identifier` parameter in the `showBioentityPage` method. This is your gene ID.

![screen shot 2018-06-21 at 14 03 11](https://user-images.githubusercontent.com/5923538/41720738-e25e0fea-755b-11e8-9b7c-d67fb2905f45.png)